test/parallel/test-icu-data-dir.js

```diff
@@ -8,7 +8,7 @@ const { spawnSync } = require('child_process');
 
 const expected =
     'could not initialize ICU (check NODE_ICU_DATA or ' +
-    '--icu-data-dir parameters)' + (common.isWindows ? '\r\n' : '\n');
+    `--icu-data-dir parameters)${common.isWindows ? '\r\n' : '\n'}`;
```

Comment:
Actually since `expected` is later used with `str.includes(expected)` you could remove the EOL completely, and fit the string in a single line.